pkg/trace/config/config.go

```diff
@@ -253,6 +252,29 @@ type DebuggerProxyConfig struct {
 	APIKey string
 }
 
+// AppSecConfig hold the AppSec reverse proxy settings.
+type AppSecConfig struct {
```

Comment:
I didn't touch it in the end because we plan to remove this part very soon.

Also, note that I couldn't put the obfuscator configuration as a field of this struct because the `Export()` method of `ObfuscationConfig` doesn't have access to it when called to create the obfuscator pacakge configuration.